Source/WTF/wtf/unicode/CharacterNames.h

```diff
@@ -45,8 +45,11 @@ constexpr UChar arabicIndicPerTenThousandSign = 0x060A;
 constexpr UChar arabicPercentSign = 0x066A;
 constexpr UChar blackCircle = 0x25CF;
 constexpr UChar blackDownPointingSmallTriangle = 0x25BE;
+constexpr UChar blackDownPointingTriangle = 0x25BC;
 constexpr UChar blackLeftPointingSmallTriangle = 0x25C2;
+constexpr UChar blackLeftPointingTriangle = 0x25C0;
 constexpr UChar blackRightPointingSmallTriangle = 0x25B8;
+constexpr UChar blackRightPointingTriangle = 0x25B6;
 constexpr UChar blackSquare = 0x25A0;
 constexpr UChar blackUpPointingSmallTriangle = 0x25B4;
 constexpr UChar blackUpPointingTriangle = 0x25B2;
```

Comment:
I just realized we can remove the small variants from this file now that we no longer use them